media-sound/teamspeak-server-bin/teamspeak-server-bin-3.1.1.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit multilib systemd user
+
+DESCRIPTION="Crystal Clear Cross-Platform Voice Communication Server"
+HOMEPAGE="https://www.teamspeak.com/"
+SRC_URI="
+	amd64? ( http://ftp.4players.de/pub/hosted/ts3/releases/${PV}/teamspeak3-server_linux_amd64-${PV}.tar.bz2 )
+	x86? ( http://ftp.4players.de/pub/hosted/ts3/releases/${PV}/teamspeak3-server_linux_x86-${PV}.tar.bz2 )"
+
+SLOT="0"
+LICENSE="teamspeak3"
+IUSE="doc tsdns"
+KEYWORDS="~amd64 ~x86"
+
+RESTRICT="installsources fetch mirror strip"
+
+S="${WORKDIR}/teamspeak3-server_linux_${ARCH}"
+
+QA_PREBUILT="opt/*"
+
+pkg_nofetch() {
+	elog "Please download ${A}"
+	elog "from ${HOMEPAGE}downloads and place this"
+	elog "file in ${DISTDIR}"
+}
+
+pkg_setup() {
+	enewuser teamspeak3
+}
+
+src_install() {
+	# Install wrapper
+	dosbin "${FILESDIR}"/ts3server
+
+	# Install TeamSpeak 3 server into /opt/teamspeak3-server.
+	local opt_dir="/opt/teamspeak3-server"
+	into ${opt_dir}
+	insinto ${opt_dir}
+	exeinto ${opt_dir}
+	newsbin ts3server ts3server-bin
+	doexe *.sh
+	doins *.so
+	doins -r sql
+
+	# Add ".ts3server_license_accepted", since LICENSE="teamspeak3" is set
+	touch ${D}/${opt_dir}/.ts3server_license_accepted
+
+	# Install documentation and tsdns.
+	dodoc -r CHANGELOG doc/*.txt
+	use doc && dodoc -r serverquerydocs doc/serverquery && \
+		docompress -x /usr/share/doc/${PF}/serverquerydocs && \
+		docompress -x /usr/share/doc/${PF}/serverquery && \
+		dosym ../../../usr/share/doc/${PF}/serverquery ${opt_dir}/doc/serverquery && \
+		dosym ../../usr/share/doc/${PF}/serverquerydocs ${opt_dir}/serverquerydocs
+
+	if use tsdns; then
+		newsbin tsdns/tsdnsserver tsdnsserver
+		newdoc tsdns/README README.tsdns
+		newdoc tsdns/USAGE USAGE.tsdns
+		dodoc tsdns/tsdns_settings.ini.sample
+	fi
+
+	# Install the runtime FS layout.
+	insinto /etc/teamspeak3-server
+	doins "${FILESDIR}"/server.conf "${FILESDIR}"/ts3db_mariadb.ini
+	keepdir /{etc,var/{lib,log}}/teamspeak3-server
```

Comment:
This is unreadable, please split it into the non-lazy variant.

Reply:
Done.